pennylane/circuit_drawer/circuit_drawer.py

```diff
@@ -55,21 +55,33 @@ def __init__(
         raw_operation_grid,
         raw_observable_grid,
         wires,
-        charset=UnicodeCharSet,
+        charset=None,
         show_all_wires=False,
     ):
         self.operation_grid = Grid(raw_operation_grid)
         self.observable_grid = Grid(raw_observable_grid)
         self.wires = wires
         self.active_wires = self.extract_active_wires(raw_operation_grid, raw_observable_grid)
-        self.charset = charset
+
+        if charset is None:
+            self.charset = CHARSETS["unicode"]()
```

Comment:
Is `CHARSETS["unicode"]()` the same as `charsets.UnicodeCharSet`?

Reply:
An instance of that, yes. Another change in this PR is that we use instances of that class instead of the class itself.

Reply:
> Another change in this PR is that we use instances of that class instead of the class itself.

Could this change have any downsides or raise potential unexpected issues, e.g. when passing it along to the representation resolver?

Reply:
nope. everything is static methods.